src/main/java/io/vertx/ext/stomp/StompClientOptions.java

```diff
@@ -81,7 +84,18 @@ public StompClientOptions(StompClientOptions other) {
    */
   public StompClientOptions(JsonObject json) {
     super(json);
+    init();
     StompClientOptionsConverter.fromJson(json, this);
+
+    // If no valid reuseAddress value specified in the json, set it with DEFAULT_CLIENT_REUSE_ADDRESS
```

Comment:
Not sure the converter inheritance can handle this, the parent NetworkOptions default to set setReuseAddress as true, but in StompClientOptions, suggest to set it default as false. Since issue #43 is hard to find out why if default to setReuseAddress as true.

Reply:
Thanks!